.github/workflows/asv-bot.yml

```diff
@@ -41,13 +41,8 @@ jobs:
 
         # Although asv sets up its own env, deps are still needed
         # during discovery process
-      - uses: conda-incubator/setup-miniconda@v2.1.1
-        with:
-          activate-environment: pandas-dev
-          channel-priority: strict
-          environment-file: ${{ env.ENV_FILE }}
-          use-only-tar-bz2: true
-          condarc-file: ci/condarc.yml
+      - name: Set up Conda
+        uses: ./.github/actions/setup-conda
```

Comment:
Nit: Could we rename this actions `setup_conda` to match `build_pandas`?

Reply:
The GHA convention seems to be `kebab-case`, see eg. https://github.com/conda-incubator/setup-miniconda/blob/main/action.yml#L23

I think we should stick to one style everywhere (action names, inputs, ...). Please make a decision if we want `snake_case` or `kebab-case`.

Reply:
Okay happy to change `build_pandas` to `build-pandas` then (feel free to change this in one of your other PRs)